controllers/update.go

```diff
@@ -298,6 +300,8 @@ func (r *ApplicationReconciler) getAndAddComponentApplicationsToModel(log logr.L
 	// Add the components to the Devfile model
 	err = r.addComponentsToApplicationDevfileModel(devSpec, components)
 	if err != nil {
+		// User error - so increment the "success" metric - since we're tracking only system errors
+		metrics.ApplicationCreationSucceeded.Inc()
```

Comment:
If we are tracking only system err why even inc this?

Reply:
Because the reconcile still "succeeded" as expected (i.e. no system errors).